wire-schema/src/main/java/com/squareup/wire/schema/Pruner.java

```diff
@@ -231,7 +231,7 @@ private void markReachable() {
 
     for (ProtoMember member : options.fields().values()) {
       // If it's an extension, don't consider the entire enclosing type to be reachable.
-      if (isExtensionField(member)) {
+      if (!isExtensionField(member)) {
```

Comment:
Tests didn’t catch this?

Reply:
The content's logic of the method has been inverted as well. This isn't really a behavior change, just testing what the method said it was.